crates/node_binding/binding.d.ts

```diff
@@ -326,12 +326,6 @@ export interface JsStatsWarning {
   formatted: string
 }
 
-export interface NodeFS {
-  writeFile: (...args: any[]) => any
-  mkdir: (...args: any[]) => any
-  mkdirp: (...args: any[]) => any
```

Comment:
It is not expected.

Reply:
it should be fixed by napi-rs, cc @IWANABETHATGUY